nodejs/awsx/elasticloadbalancingv2/listenerRule.ts

```diff
@@ -33,9 +33,11 @@ export class ListenerRule extends pulumi.ComponentResource {
 
     constructor(name: string, listener: x.elasticloadbalancingv2.Listener,
                 args: ListenerRuleArgs, opts: pulumi.ComponentResourceOptions = {}) {
-        super("awsx:x:elasticloadbalancingv2", name, {}, { parent: listener, ...opts });
+        // We forgot to add the `ListenerRule` part of the name.  Add it in and create an alias from
+        // the previous incorrect name.
+        const aliases = [pulumi.createUrn(name, "awsx:x:elasticloadbalancingv2", opts.parent)];
+        super("awsx:x:elasticloadbalancingv2:ListenerRule", name, {}, { parent: listener, aliases, ...opts });
```

Comment:
This `aliases, ...opts` pattern is repeated quite a bit, and unfortunately I don't think it's quite safe.  It's unfortunate that there's not an easier way to express this - but I think for now we need to be a little more careful in merging these.

Reply:
You're correct.  Have switched to merging.